pkg/secretless/config/v1/config.go

```diff
@@ -0,0 +1,169 @@
+package v1
+
+import (
+	"fmt"
+	"github.com/go-ozzo/ozzo-validation"
+	"gopkg.in/yaml.v2"
+	"regexp"
+	"strconv"
```

Comment:
Sort imports consistently:
```
import (
    <stdlib>

    <go.mod imports>

    <local imports>
)
```

Reply:
Glad you pointed this out.  My IDE was set to `gofmt` style imports (which are alphabetical).  I changed to `goimports` style and it used the above formatting.